canonical-web-and-design/cn.ubuntu.com
# Liam's 20.04 updates

## Done

- Fixed up Liam's branch

## QA

1. download this repo
2. test the updated pages

@zprood can you see if this is closer to what you want?
